frontend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "badge-frontend",
+  "version": "0.0.0",
+  "description": "Shields.io frontend",
+  "private": true,
+  "homepage": "http://shields.io",
+  "license": "CC0-1.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/badges/shields.git",
+    "directory": "frontend"
+  },
+  "scripts": {
+    "test": "echo 'Run tests from parent dir'; false"
+  },
+  "devDependencies": {
+    "gatsby": "*"
```

Comment:
As I briefly explained in one of my previous messages:
> You still need to specify the Babel config and the gatsby dependency in the child package.json (otherwise Gatsby will tell you it's not a Gatsby project).

Here is the error message you get is Gatsby is not specified:
> gatsby <build> can only be run for a gatsby site.
Either the current working directory does not contain a valid package.json or 'gatsby' is not specified as a dependency